### PR TITLE
kubernetes-addons: cluster/saltbase/salt/kube-addons changed to cluster/addons/addon-manager

### PR DESCRIPTION
**Pull request by ingvagabund**
cluster/saltbase/salt/kube-addons/namespace.yaml -> cluster/addons/addon-manager/namespace.yaml

### CODE REVIEW

**Comment by ingvagabund** [comment]
Caused by https://github.com/kubernetes/kubernetes/pull/23600

**Comment by ingvagabund** [comment]
Entire directory has changed: ```cluster/saltbase/salt/kube-addons``` -> ```cluster/addons/addon-manager```

**Comment by adamschaub** [comment]
+1, this works for me thanks for the patch

**Comment by eparis** [comment]
@ingvagabund let me know when you do not think this is a WIP

**Comment by ingvagabund** [comment]
@eparis right now

File: ansible/roles/kubernetes-addons/tasks/main.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 - name: Make sure the system services namespace exists
   get_url:
-    url=https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/cluster/saltbase/salt/kube-addons/namespace.yaml
+    url=https://raw.githubusercontent.com/kubernetes/kubernetes/master/{{ kube_addons_download_source_dir }}/namespace.yaml
     dest="{{ kube_config_dir }}/addons/"
```
Comment (gouyang) [inline ansible/roles/kubernetes-addons/tasks/main.yml]: dest is hard coding to /opt/namespace.yaml in kube-addons.sh, it need to adjust accordingly.

https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/addon-manager/kube-addons.sh#L68

Comment (ingvagabund) [inline ansible/roles/kubernetes-addons/tasks/main.yml]: @mikedanese , WDYT? Maybe kube-addons.sh.j2 and replace the /opt/namespace.yaml with {{ kube-addons-namespace-location }}/namespace.yaml or similar buzzword?

Comment (mikedanese) [inline ansible/roles/kubernetes-addons/tasks/main.yml]: Why can't you just run addon manager in a static pod? that file exists inside gcr.io/google-containers/kube-addon-manager

Comment (ingvagabund) [inline ansible/roles/kubernetes-addons/tasks/main.yml]: I have no idea. Maybe, it is "just" a coincidence and the /opt/namespace.yaml path is correct.

**Pull request merged**